hooks/pre/13-clear_pulpcore_content.rb

```diff
@@ -0,0 +1,18 @@
+# See bottom of the script for the command that kicks off the script
+require 'English'
+
+def reset
+  stop_services
+  start_services(['postgresql']) if local_postgresql?
+  empty_db_in_postgresql('pulpcore') if pulpcore_enabled?
+end
+
+if app_value(:clear_pulpcore_content) && !app_value(:noop)
+  response = ask('Are you sure you want to continue? This will clear all Pulpcore content including repositories and synced content. [y/n]')
```

Comment:
It doesn't remove synced content, only the database. Not sure it matters to end users, but they may be surprised to find the disk usage is still high.

Reply:
I took inspiration from (read: shamelessly copied) @ianballou 's work on https://github.com/theforeman/foreman-installer/pull/504 into a new helper in `hooks/boot/05-data-helpers.rb` and incorporated it in `hooks/pre/13-clear_pulpcore_content.rb`